ent/template/mutation_input.tmpl

```diff
@@ -10,130 +10,150 @@
 {{ template "import" $ }}
 
 {{- range $n := $.Nodes }}
-    {{ $input := print "Create" $n.Name "Input" }}
-    // {{ $input }} represents a mutation input for creating {{ plural $n.Name | lower }}.
-    type {{ $input }} struct {
-        {{- range $f := $n.Fields }}
-            {{- if not $f.IsEdgeField }}
-                {{ $f.StructField }} {{ if and (or $f.Optional $f.Default) (not $f.Type.RType.IsPtr) }}*{{ end }}{{ $f.Type }}
+    {{- $annotation := $n.Annotations.EntGQL }}
+    {{- if or (not $annotation) (not $annotation.Skip) }}
```

Comment:
Just note that templates in Go 1.17 do not respect short-circuit. I'll be added in 1.18 (together with `continue` and `break`)